man/systemd.exec.xml

```diff
@@ -303,6 +303,29 @@
       </varlistentry>
 
       <varlistentry>
+        <term><varname>PassEnvironment=</varname></term>
+
+        <listitem><para>Pass PID1 environment variables for executed processes.
+        Takes a space-separated list of variable names. This option may be
+        specified more than once, in which case all listed variables will be
+        set. If the empty string is assigned to this option, the list of
```

Comment:
Environment variables from `Environment` and `EnvrionmentFile` have a higher priority:
```
# systemctl show -p Environment
Environment=A=1 LANG=C.UTF-8 PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin

# systemctl cat pass-env
# /etc/systemd/system/pass-env.service
[Service]
Type=oneshot
Environment=A=unit
PassEnvironment=A B C
Environment=B=2
ExecStart=/usr/bin/env

# systemctl start pass-env

# journalctl -u pass-env -b
-- Logs begin at Sat 2015-11-07 11:42:38 UTC, end at Sat 2015-11-07 13:21:50 UTC. --
Nov 07 13:21:50 wily-64 systemd[1]: Starting pass-env.service...
Nov 07 13:21:50 wily-64 env[49]: A=unit
Nov 07 13:21:50 wily-64 env[49]: LANG=C.UTF-8
Nov 07 13:21:50 wily-64 env[49]: PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
Nov 07 13:21:50 wily-64 env[49]: B=2
Nov 07 13:21:50 wily-64 systemd[1]: Started pass-env.service.
```
I think, the manpage should contain that info.